packages/paste-core/components/detail-text/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@twilio-paste/detail-text",
+  "version": "0.0.0",
+  "category": "typography",
+  "status": "production",
+  "description": "Detail text is typography used for small, short pieces of secondary text content.",
+  "author": "Twilio Inc.",
+  "license": "MIT",
+  "main:dev": "src/index.tsx",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn clean && NODE_ENV=production node build.js && tsc",
+    "build:js": "NODE_ENV=development node build.js",
+    "build:props": "typedoc --tsconfig ./tsconfig.json --json ./dist/prop-types.json",
+    "clean": "rm -rf ./dist",
+    "tsc": "tsc"
+  },
+  "peerDependencies": {
+    "@twilio-paste/animation-library": "^0.3.2",
+    "@twilio-paste/box": "^7.0.0",
+    "@twilio-paste/customization": "^5.0.0",
+    "@twilio-paste/design-tokens": "^8.0.0",
+    "@twilio-paste/style-props": "^6.0.0",
+    "@twilio-paste/styling-library": "^1.0.0",
+    "@twilio-paste/theme": "^8.0.0",
+    "@twilio-paste/types": "^3.1.1",
+    "prop-types": "^15.7.2",
+    "react": "^16.8.6 || ^17.0.2",
+    "react-dom": "^16.8.6 || ^17.0.2"
+  },
+  "devDependencies": {
+    "@twilio-paste/animation-library": "^0.3.2",
```

Comment:
question: why do we need animation library?

Reply:
Because box is dependent on it - we have to have all the nested dependencies listed.

Reply:
does this cause issues if we accidentally don't include nested dependencies? I was looking through other component package.json files and other packages list box but not animation library, like Card for example. Not related to this PR, but is that something we should fix and/or make a danger.js rule for?

Reply:
Yeah, I think it was fixed by this: https://github.com/twilio-labs/paste/pull/2838 - but maybe @Niznikr  @TheSisb have more context

Reply:
Right, ESLint correctly catches these issues.